### ISSUE TITLE
possibly not compatible with other cmp

### ISSUE THREAD

**Issue by fleongg** [ayamir/nvimdots#239]
**Describe the bug**

I install [copilot.lua](https://github.com/zbirenbaum/copilot.lua) and [copilot-cmp.lua](https://github.com/zbirenbaum/copilot-cmp). It should show copilot suggestions on completion windows. However there is no copilot completions.

**To Reproduce**
1. add the following code to the completion/config.lua file:
```
function config.copilot()
	local defer = vim.defer_fn(function()
		require("copilot").setup({
			panel = { -- no config options yet
				enabled = true,
			},
			ft_disable = {},
			copilot_node_command = "node", -- Node version must be < 18
			plugin_manager_path = vim.fn.stdpath("data") .. "/site/pack/packer",
			server_opts_overrides = {},
		})
	end, 100)
end

function config.copilot_cmp()
	require("copilot_cmp").setup({
		method = "getCompletionsCycling",
		force_autofmt = false,
		formatters = {
			label = require("copilot_cmp.format").format_label_text,
			insert_text = require("copilot_cmp.format").format_insert_text,
			preview = require("copilot_cmp.format").deindent,
		},
	})
end
```

2. add the following to the completion/plugins.lua file:

```
completion["zbirenbaum/copilot.lua"] = {
	opt = true,
	event = "VimEnter",
	config = conf.copilot,
}

completion["zbirenbaum/copilot-cmp"] = {
	opt = true,
	after = { "copilot.lua", "nvim-cmp" },
	config = conf.copilot_cmp,
}
```

Or, I tried the method from https://github.com/zbirenbaum/copilot-cmp/issues/32 by adding the two lines:
<img width="771" alt="image" src="https://user-images.githubusercontent.com/88166834/193174913-f5102989-4040-45eb-b8b2-d1daa559cb4b.png">

still nothing happens.

<img width="1009" alt="image" src="https://user-images.githubusercontent.com/88166834/193175245-fdaf4897-e51f-4630-a90c-521867f9dbf1.png">

**Environment (please complete the following information):**
 - OS: [macOS 12.6]
 - Nvim version(output of `nvim --version`)
```
NVIM v0.7.2
Build type: Release
LuaJIT 2.1.0-beta3
Compiled by brew@HMBRW-A-001-M1-004.local

Features: +acl +iconv +tui
See ":help feature-compile"

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/opt/homebrew/Cellar/neovim/0.7.2_1/share/nvim"

Run :checkhealth for more info
```

**Comment by fleongg** [comment]
Also, the copilot plugin is not showing inline suggestions with the default config

**Comment by ayamir** [comment]
fixed at 0369c26f21d34d4291d85080d1c32ca77512fb5d

**Comment by fleongg** [comment]
Thanks a lot! this is great!
